ts/src/grpc.ts

```diff
@@ -102,17 +124,46 @@ export namespace grpc {
     return new Uint8Array(frame);
   }
 
-  function getStatusFromHeaders(headers: BrowserHeaders): Code {
+  function getStatusFromHeaders(headers: BrowserHeaders): Code | null {
     const fromHeaders = headers.get("grpc-status") || [];
     if (fromHeaders.length > 0) {
       try {
         const asString = fromHeaders[0];
         return parseInt(asString, 10);
       } catch (e) {
-        return Code.Internal;
+        return null;
       }
     }
-    return Code.Internal;
+    return null;
+  }
+
+  export function unary<TRequest extends jspb.Message, TResponse extends jspb.Message, M extends UnaryMethodDefinition<TRequest, TResponse>>(methodDescriptor: M,
+                                                                                                                                        props: UnaryRpcOptions<M, TRequest, TResponse>) {
```

Comment:
It's the formatting. The line has the rest of the parameters on, but they're off-screen on GH. I'll just put them all on the line above.